manim/mobject/svg/text_mobject.py

```diff
@@ -373,7 +373,7 @@ def construct(self):
                 arb = Text(
                     "صباح الخير \n تشرفت بمقابلتك", font="sans-serif"
                 )  # don't mix RTL and LTR languages nothing shows up then ;-)
-                japanese = Text("臂猿「黛比」帶著孩子", font="sans-serif")
+                chinese = Text("臂猿「黛比」帶著孩子", font="sans-serif")
                 self.add(morning,chin,mess,russ,hin,arb,japanese)
```

Comment:
Yikes, I didn't notice we had Chinese and Japanese swapped in the library.

```suggestion
                self.add(morning, chinese, mess, russ, hin, arb, japanese)
```